Week1/credit/credit.c

```diff
@@ -0,0 +1,85 @@
+#include <cs50.h>
+#include <stdio.h>
+
+int main(void)
+{
+    // input creditcard number
+    long n = get_long("Creditcardnumber: ");
```

Comment:
```suggestion
    long creditcardnumber = get_long("Creditcardnumber: ");
```

Just renaming the variable makes the comment above obsolete ;)